src/basic/copy.c

```diff
@@ -372,7 +373,12 @@ static int fd_copy_symlink(
         if (r < 0)
                 return r;
 
-        if (symlinkat(target, dt, to) < 0)
+        if (copy_flags & COPY_MAC_CREATE)
+                mac_selinux_create_file_prepare_at(dt, to, S_IFLNK);
```

Comment:
hmm, we should really propagate errors here, no?

Reply:
done in split out pr